DataManagementSystem/Client/FTS3Job.py

```diff
@@ -51,14 +55,13 @@ def __init__(self):
     self.error = None
     self.status = FTS3Job.INIT_STATE
 
-
     self.completeness = None
-
     self.operationID = None
 
     self.username = None
     self.userGroup = None
 
+    self.vo = None
```

Comment:
This should go a bit lower, with all the other temporary attributes set by FTS Operation no ?